cfme/containers/node.py

```diff
@@ -238,28 +238,31 @@ class Utilization(CFMENavigateStep):
     prerequisite = NavigateToSibling('Details')
 
     def step(self):
-        self.prerequisite_view.monitor.item_select('Utilization')
+        """Navigate to the Utilization page"""
+        self.prerequisite_view.toolbar.monitoring.item_select('Utilization')
 
 
-class NodeTimelinesForm(NodeView):
-    # TODO PR 3710, timeline widget
+class NodeTimelinesView(NodeView, TimelinesView):
+    """Timeline page for Nodes"""
+    VIEW = TimelinesView
```

Comment:
**REQUIRED**: Use the `Ignore` decorator here to avoid the TimelinesView being instantiated automatically if you intend this to be a constant. Though this looks to me like some sort of an artifact copied over from a step.

Reply:
:+1: done